src/Core/Configure.php

```diff
@@ -342,6 +342,41 @@ public static function dump($key, $config = 'default', $keys = [])
     }
 
     /**
+     * Write new and overwrite existing values into an existing configuration file.
+     *
+     * ### Usage
+     *
+     * ```
+     * Configure::updateDump('my_config', 'default', ['Key.key' => 'value'])
+     * ```
+     *
+     * The third parameter ($append) follows the same syntax as Configure::write()
+     * but as an array of values to be created or updated.
+     *
+     *
+     * @param string $key Identifier of the config file to be updated.
+     * @param string $config  The name of the configured adapter to dump data with.
+     * @param array $append Values to add or overwrite
+     */
+    public static function updateDump($key, $config = 'default', array $append = [])
```

Comment:
Having `$append` as empty array by default doesn't make much sense. It would mean just rewriting the config file without any change. Also `$append` isn't a very good name since it would also overwrite values.

Having arguments as `updateDump(array $values, $key, $config = 'default')` makes more sense to me.